python/pyspark/sql/pandas/group_ops.py

```diff
@@ -329,8 +327,6 @@ def applyInPandasWithState(
         Notes
         -----
         This function requires a full shuffle.
-
-        This API is experimental.
```

Comment:
cc @HeartSaVioR can we remove this for streaming ones?

Reply:
Yeah good to remove as we passed through one minor release (3.5) and there is no outstanding issue.